src/components/ItemDetail/use/borrow.ts

```diff
@@ -1,20 +1,20 @@
 import { ref, Ref, ComputedRef, computed } from 'vue'
-import apis, { ItemSummary, ItemType, Owner, LogType } from '/@/lib/apis'
+import apis, { ItemType, Owner, LogType, ItemDetail, Log } from '/@/lib/apis'
 import useOwners, { OwnerDetail } from './owners'
 import { stringifyDate } from '/@/lib/date'
 import useMe from '/@/use/me'
 import { useStore } from '/@/store'
 
 const useBorrow = (props: {
-  item: ItemSummary
+  item: ItemDetail
 }): {
   details: ComputedRef<OwnerDetail[]>
   selectedOwnerName: Ref<string>
   purpose: Ref<string>
   dueDate: Ref<Date>
   count: Ref<number>
   owner: ComputedRef<Owner | undefined>
-  borrow: () => Promise<void>
+  borrow: () => Promise<void | Log>
```

Comment:
```suggestion
  borrow: () => Promise<Log | undefined>
```
`void`とのunionになるの微妙くない？

Reply:
throw errorの方がいい？
それとも使ってないしboolean返した方がいい？

Reply:
それ悩んだけど`undefined`とのunionでいいと思う